en/flight_controller/README.md

```diff
@@ -21,6 +21,8 @@ This list is not exhaustive - there are other compatible flight controllers and
 - [Manufacturer-Supported Autopilots](../flight_controller/autopilot_manufacturer_supported.md)
   - [AirMind MindPX](../flight_controller/mindpx.md)
   - [AirMind MindRacer](../flight_controller/mindracer.md)
+  - [CUAV X7](../flight_controller/cuav_x7.md)
```

Comment:
FYI, I removed a link to the cuav hardware repo as it was inconsistent. That info should be added (by you) if needed in the docs themselves.